packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts

```diff
@@ -59,8 +59,26 @@ describe('Token', () => {
     expect(tokenBalances[0]).toStrictEqual('5.00000000@1')
   })
 
+  it('should mintTokens to an address', async () => {
```

Comment:
Can we also cover test case for minting tokens by a consortium? just to make sure it's accounted in the consortium's `minted` and `daily minted` if a consortium execute it

Reply:
Nice idea. Added the test in https://github.com/JellyfishSDK/jellyfish/pull/1965/commits/e0ce2ff6b3551ea698bc5bb73eb7c46357736311